packages/astro/src/runtime/server/render/component.ts

```diff
@@ -110,21 +123,9 @@ export async function renderComponent(
 		metadata.componentExport = hydration.componentExport;
 		metadata.componentUrl = hydration.componentUrl;
 	}
-	const probableRendererNames = guessRenderers(metadata.componentUrl);
-
-	if (
-		Array.isArray(renderers) &&
-		renderers.length === 0 &&
-		typeof Component !== 'string' &&
-		!componentIsHTMLElement(Component)
-	) {
-		const message = `Unable to render ${metadata.displayName}!
-
-There are no \`integrations\` set in your \`astro.config.mjs\` file.
-Did you mean to add ${formatList(probableRendererNames.map((r) => '`' + r + '`'))}?`;
-		throw new Error(message);
-	}
```

Comment:
Due to the hidden `astro:jsx` renderer, this was never true. I first tried to get it working, but there was a lot of edge cases that I couldn't figure out, so I just removed it since there's another error down below that covers the same case (No renderer found for..)